plugins/woocommerce/tests/php/includes/emails/class-wc-email-customer-pos-refunded-order-test.php

```
@@ -403,6 +403,42 @@ public function test_email_content_includes_refund_returns_policy_when_option_is
 		$this->assertStringContainsString( 'Accepted within 30 days of purchase.', $plain_text_content );
 	}
 
+	/**
+	 * @testdox POS refunded email displays unit price in the quantity column.
+	 */
+	public function test_pos_email_displays_unit_price_in_quantity_column() {
+		// Initialize WC_Emails to set up actions and filters.
+		$emails = new WC_Emails();
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$emails'. (undefined)

(UnusedLocalVariable)